src/summaries/units_variables.yaml

```diff
@@ -2,15 +2,194 @@
   shortTitle: Units
   key: "units"
   details: |
-    - time units
-    - ether units
+    - Ether Units
+      * In Solidity, we can use wei, gwei or ether to specify a subdenomination of Ether.
+      * Lowest unit is wei and $1e12$ represents $1 * 10^{12}$.
+      * `1 wei = 1;`
+      * `1 gwei = 1e9;`
+      * `1 ether = 1e18;`
+
+    - Time Units
+      * We can use seconds, minutes, hours, days and weeks as suffix to denote time.
+      * Lowest unit of time is `seconds`.
+      * `1 seconds = 1;`
+      * `1 minutes = 60 seconds;`
+      * `1 hours = 60 minutes;`
+      * `1 day = 24 hours;`
+      * `1 week = 7 days;`  
 
 - title: Globally available Variables
   shortTitle: Global Variables
-  key: "variables"
+  key: "global-variables"
+  details: |
+    - There are special variables and functions which always exist in the global namespace and are mainly used to provide 
+      information about the blockchain or are general-use utility functions.
+    - These are of various types having specific properties.
+      * Block and Transaction properties.
+      * ABI Encoding and Decoding functions.
+      * Members of bytes and string.
```

Comment:
Can you please add sections for 
     * Members of bytes and string.
     * Error handling methods.
     *  Members of address type
 Or if the content is less, then combine these into one sub-section